services/io_services/io_read_pokemon.go

```diff
@@ -0,0 +1,56 @@
+package io_services
+
+import (
+	"encoding/csv"
+	"fmt"
+	"jmc/bootcamp/models"
+	"os"
+)
+
+type CsvService struct {
+  PathFile string
+}
+
+func (srv CsvService)ReadFromService() (pokemonList []models.Pokemon, err error) {
+
+	csvFile, err := os.Open(srv.PathFile)
+	if err != nil {
+		return nil, fmt.Errorf("error try to open the CSV file (%v): %v", srv.PathFile, err)
+	}
+
+	defer csvFile.Close()
```

Comment:
You can mange the possible error that `Close()` returns implementing something like this: 
```suggestion
defer func(){
  err := csvFile.Close()
  if err != nil {
    log.Errorf("file cannot be closed: %v", err)
  }
}
```